src/main/java/ecdar/simulation/SimulationState.java

```diff
@@ -1,51 +1,42 @@
 package ecdar.simulation;
 
 import EcdarProtoBuf.ObjectProtos;
+import EcdarProtoBuf.ObjectProtos.State;
+import ecdar.Ecdar;
 import javafx.util.Pair;
 
-import java.math.BigDecimal;
 import java.util.ArrayList;
 
 public class SimulationState {
+    // key = component name, value = id
```

Comment:
comment?

Reply:
jaer det var for at forklare at de bliver gemt som et key-value pair hvor key repræsentarer component name og value er id på edge/location